source/game/g_weapon.cpp

```diff
@@ -566,16 +566,18 @@ static void G_Fire_SunflowerPattern( edict_t *self, vec3_t start, vec3_t dir, in
 		}   
 	} 
 
- G_ApplyHandicapDamage(self,&damage);
+ 	G_ApplyHandicapDamage(self,&damage);
     
      for( int i = 1; i <= MAX_CLIENTS; i++ ) {
 		if( hits[i] == 0 )
 			continue;
+		if (!GS_Instagib()){
 		edict_t * target = &game.edicts[i];
 		edict_t * ev = G_SpawnEvent( EV_DAMAGE, 0, target->s.origin );
 		ev->r.svflags |= SVF_OWNERANDCHASERS;
 		ev->s.ownerNum = ENTNUM( self );
 		ev->s.damage = HEALTH_TO_INT( hits[i] * damage );
+		}
```

Comment:
could you indent this correctly.

Reply:
I didn't know that these changes would end up in an existing commit, very sorry! Should I revert these changes in existing branch, or submit new PR specifically with changes related to aarch64 fixes? I'm really sorry, I'm still learning how to use git

Reply:
you can just reset the head and push or interactive rebase

git reset HEAD~1
git push <your-branch> --force

Reply:
Well, I removed needed commit by mistake. Added it again, feeling stupid 🙈

Reply:
its still there you just closed it.